e2e/build_uki_test.go

```diff
@@ -52,7 +58,9 @@ var _ = Describe("build-uki", func() {
 	})
 
 	It("successfully builds a uki iso from a container image", func() {
-		out, err := enki.Run("build-uki", image, "--output-dir", resultDir, "-k", keysDir, "--output-type", "iso")
+		out, err := enki.Run("genkey", "--output", "assets/keys", "TEST")
```

Comment:
doesn't this overwrite the existing ones?

Reply:
Never mind, I saw the `RemoveAll` above now. That's why the other test is failing. You need to create the keys in the test above this one too.

Reply:
:facepalm:

Reply:
you need to pass `keysDir` instead of `assets/keys` here. Remember, enki is running in side a container and the additional directory is mounted as a volume: https://github.com/kairos-io/enki/blob/87c56d69b671fd41e445a81472b576420781d5da/e2e/suite_test.go#L46